docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss

```diff
@@ -28,6 +28,10 @@
     color: darken($su-color-driftwood, 45%);
     font-size: hb-calculate-rems(14px);
     font-style: italic;
+
+    .hb-local-footer--dark & {
```

Comment:
hmmm I'm trying to decide whether these styles should just live in the footer.scss.  I think it might make more sense

Reply:
I was wondering about that too. I chose to leave it here in case other components with dark background colors need the same styles, all we would need to do is to add the container class. For example, 
```
.hb-local-footer--dark,
.hb-another-component--dark {
    color: $hb-color--white;
}
```

What do you think? I'm still happy to move the styles in the footer as I can see how it makes sense both ways.

Reply:
I can imagine scenarios where we'll need to update styles for other themes and keeping all of the WYSIWYG styles together will ensure that nothing gets missed in the update.

If we do want to pull the styles into the local footer partial we'll need to create some mixins to ensure consistency, especially if we have other dark themed components like Heather suggested. Is that something that would improve code readability, will it make it easier to extend the code in the future, and is it within the scope of this card?

Reply:
I'm fine to leave these in the WYSIWYG styles! That's good to know, I think there's places in the code in the future I'll want to refactor where I overrode a WYSIWYG style.

Reply:
I decided against refactoring at the moment. We currently have a card [STN-193](https://sparkbox.atlassian.net/browse/STN-193) in the backlog for refactoring the "decanter" WYSIWYG classes. A refactor would be better suited for that card.